doc/TRAINING.rst

```diff
@@ -12,11 +12,13 @@ Prerequisites for training a model
 Getting the training code
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Clone the DeepSpeech repository:
+Clone the latest released stable branch from Github (e.g. 0.8.2, check `here <https://github.com/mozilla/DeepSpeech/releases>`_ for current version) to start with as the master branch is under active development and might produce errors: 
```

Comment:
nit: "the master branch provides no guarantee regarding its workable state, though we are trying to keep it as stable as possible ; breakages can happen and feedback is welcome."